src/coreclr/vm/dynamicmethod.cpp

```diff
@@ -448,7 +443,7 @@ HeapList* HostCodeHeap::InitializeHeapList(CodeHeapRequestInfo *pInfo)
     else
 #endif // FEATURE_INTERPRETER
     {
-#if defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64)
+#if defined(TARGET_64BIT) && defined(TARGET_WINDOWS)
```

Comment:
`if (pInfo->IsInterpreted())` block needs to be moved inside this ifdef to fix the build break